server/@types/supportAdditionalNeedsApi/index.d.ts

```diff
@@ -532,6 +532,12 @@ export interface components {
        * @example BXI
        */
       updatedAtPrison: string
+      /**
```

Comment:
Re-import the swagger spec, to get the new alnScreenerDate fields (plus other new stuff since last time we imported the spec 😁 )